src/components/CharactersPage/CharactersTable/CharactersTable.js

```diff
@@ -86,11 +94,59 @@ const CharactersTable = ({ onRowClick }) => {
     },
   ]
 
+  const handlePageChange = (event, newPage) => {
+    if (newPage > tablePage) {
+      onFetchMore({
```

Comment:
I actually wouldn't call `fetchMore` here, instead you should refetch since you are always making a network request and replacing the data in the cache. As the name suggests `fetchMore` should be used when you are adding to the data you already have